internal/test/envtest/environment.go

```diff
@@ -147,8 +151,28 @@ func Run(ctx context.Context, input RunInput) int {
 		return input.M.Run()
 	}
 
+	// Calculate the scheme.
+	scheme := runtime.NewScheme()
+	utilruntime.Must(appsv1.AddToScheme(scheme))
```

Comment:
Would it make sense to add a func here to dedup this a little bit with what we do in init? (pass in the scheme and register all the packages that we need in both cases)

Reply:
Does only work for the CAPI apis (because scheme.Scheme already has corev1 etc.). But will do for them.